sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplate.java

```diff
@@ -16,16 +14,23 @@
 
 import java.util.*;
 
+
 @Setter
 @Getter
+@AllArgsConstructor
 @NoArgsConstructor
+@EqualsAndHashCode
+@Builder
+@ToString
```

Comment:
> what is the result of toString() on DaTypeName and DoTypeName ?

`toString()` is defined in their super class `DataTypeName`


> since the use of toString is to check if all field are not blank why not to use a dedicated method ? It will be more comprehensive

The use of `toString()` is to log and simplify test and debug.

Reply:
> The use of toString() is to log and simplify testing and debugging.

Well yes but : 

- of course it help for easilly read tests, but that does not mean we want to commit it ?
- debugging : for debugging we should use the debug feature don't you think ?
- here I can see that in the production code, the use of toString is to check if all field are not blank, so for now it is not used in logging ? Maybe I'm mistaken

Reply:
OK